src/manifestManager.ts

```diff
@@ -186,9 +186,14 @@ export class ManifestManager implements vscode.Disposable {
     async selectManifest(): Promise<Manifest | null> {
         const quickPickItems: ManifestQuickPickItem[] = []
         const manifests = await this.getManifests()
+        const activeManifest = this.getActiveManifest()
+
         manifests.forEach((manifest) => {
+            const label = manifest.uri.fsPath === activeManifest?.uri.fsPath
+                ? `${manifest.id()} (Currently active)` : manifest.id()
```

Comment:
We can't use some emoji here like ✔️?

Reply:
![image](https://user-images.githubusercontent.com/64297935/155882516-a7573ff8-e57f-49bb-8acc-d53aa8bf17e5.png)
Possible, but seems not very visible though

Reply:
![image](https://user-images.githubusercontent.com/64297935/155882652-201a27e6-3eb6-44ac-8821-403beac9f8a9.png)

Reply:
If the id is too long then you probably won't notice the check, so probably putting the check before the id is a better option?

Reply:
The alignment for me feels weird when it comes before the id

Reply:
Can't find bold text, but this seems good to compensate for the alignment

![image](https://user-images.githubusercontent.com/64297935/155940260-d1c20487-dbb0-4107-9d9b-4b4a1636f336.png)

Reply:
Looks good to me! Regarding making that text bold, can't you use plain html tags there?

Reply:
seems not by putting <b> tags